lucida/djinntonic/tools/install-protobuf.sh

```diff
@@ -2,17 +2,17 @@
 export PROTOBUF_VERSION=2.5.0
 
 if [ -z "$THREADS" ]; then
-  THREADS=4
+  THREADS=`nproc`
 fi
 
 if [ -d protobuf-$PROTOBUF_VERSION ]; then
```

Comment:
Yeah, that's what I meant with my description above - when I just create the directories to circumvent those errors it compiles, but afterwards it still won't run. So you should be able to fix those things the way you proposed and then the big question is just why it still doesn't run. That's the one thing I haven't been able to determine in the time I had for looking over it.